### PR TITLE
readme fixup and .gitignore additions for pycharm

### LINKED ISSUE #22: Remove Sourceforge support references
When executing spell which fail, the user is presented with output to the now defunct sf.net project.
The codebase should be updated to point to its current home.

### PR DESCRIPTION
**Pull request by psi29a**
This should help get people developing in the right direction.

Closes #22

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,15 +20,18 @@ Download
 --------
 
 Get PyFFI from
-`Sourceforge <http://sourceforge.net/projects/pyffi/files/>`_,
+`Sourceforge <https://github.com/niftools/pyffi/releases>`_,
```
Comment (neomonkeus) [inline README.rst]: Github

File: README.rst [hunk 2]
```diff
@@ -20,15 +20,18 @@ Download
...
 or install it with::
 
     easy_install -U PyFFI
 
+or::
+    pip3 install PyFFI
+
 To get the latest (but possibly unstable) code, clone PyFFI from its
 `Git repository <http://github.com/amorilia/pyffi>`_::
```
Comment (neomonkeus) [inline README.rst]: Update to niftools

File: README.rst [hunk 3]
```diff
@@ -42,7 +45,7 @@ Examples
 --------
 
 * The `Blender NIF Scripts
```
Comment (neomonkeus) [inline README.rst]: Theis repo has been renamed so link text should reflect it - Blender Nif Plugin

**Review by neomonkeus — CHANGES_REQUESTED** [review]

**Comment by psi29a** [comment]
Should be covered.

**Pull request merged**